Tests/Cosmos.Compiler.Tests.Bcl/Kernel.cs

```diff
@@ -42,10 +42,11 @@ protected override void Run()
                 BitConverterTest.Execute();
                 UnsafeCodeTest.Execute();
                 DelegatesTest.Execute();
+                EncodingTest.Execute();
                 RandomTests.Execute();
 
-                System.Collections.Generic.ListTest.Execute();
-                System.Collections.Generic.QueueTest.Execute();
+                //System.Collections.Generic.ListTest.Execute();
+                //System.Collections.Generic.QueueTest.Execute();
```

Comment:
Why are these tests commented?

Reply:
I've simply forgotten to re-enable them, I do this now.